pkg/plugin/bundle.go

```diff
@@ -31,6 +31,33 @@ type bundle struct {
 	deprecateWarning         string
 }
 
```

Comment:
The changes in Kubebuilder CLI can still.
I mean, we can use the method here .
We just need to have the old one available for those that import and use Kubebuilder as a LIB.